File: app/src/main/java/com/finger/hsd/model/ProductType_v.java
package com.finger.hsd.model;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

public class ProductType_v  extends RealmObject {
   @PrimaryKey
    private String _id;
    private String barcode;
    private String name;
    private String image;
    private Integer createAt;
    private Integer v;



    public String getId() {
        return _id;
    }

    public void setId(String id) {
        this._id = id;
    }

    public String getBarcode() {
        return barcode;
    }

    public void setBarcode(String barcode) {
        this.barcode = barcode;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public Integer getCreateAt() {
        return createAt;
    }

    public void setCreateAt(Integer createAt) {
        this.createAt = createAt;
    }

    public Integer getV() {
        return v;
    }

    public void setV(Integer v) {
        this.v = v;
    }


}

File: app/src/main/java/com/finger/hsd/model/ProductType_v.kt
package com.finger.hsd.model

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import java.io.Serializable

open class ProductType_v : RealmObject(), Serializable {
    @PrimaryKey
    var id: String?= null
    var barcode: String?= null
    var name: String?= null
    var image: String? = null
    var check_product : Boolean = false // Kiểm duyệt sản phẩm
    var check_barcode : Boolean = false // kiểm tra có phải là barcode thật hay không. Có một số sản phẩm không có barcode.
    var create_at: Int? = null
    var v: Int? = null


}
